### ISSUE TITLE
CustomHeaderComponent with borderRadius

### ISSUE THREAD

**Issue by ex-git** [ammarahm-ed/react-native-actions-sheet#52]
When using CustomHeaderComponent with borderRadius set > 9 the default white background will show behind the custom header component.
I want to change the background color of the header component. Is there a props to set that?

**Comment by ammarahm-ed** [comment]
I think you can do it with `containerStyle` and set the background color of the main container or change the borderRadius so it matches with the header component.

**Comment by ex-git** [comment]
I can't get it to work. I can set background color in containerStyle. But borderRadius won't go over 9. Below example code still get radius like 9. 

containerStyle={{
    borderRadius: 100, // big number just for example
}}

**Comment by ammarahm-ed** [comment]
fixed in v0.4.0

**Comment by anatooly** [comment]
Border radius change only:
```
 containerStyle={{
        borderTopRightRadius: 15,
        borderTopLeftRadius: 15,
      }}
  ```
  that not worked:
  ```
 containerStyle={{
        borderRadius: 15,
      }}
  ```
